### PR TITLE
Fix wget in linux update guide

### LINKED ISSUE #2857: Updating on Linux guide `wget` command doesn't work
On the [linux update page](https://freshrss.github.io/FreshRSS/en/admins/07_LinuxUpdate.html), there's a command to download the zip file:

`wget -o freshrss.zip https://github.com/FreshRSS/FreshRSS/archive/1.15.3.zip
unzip freshrss.zip`

This however doesn't work because the `-o` flag in wget isn't the same as curl's.  For wget it's:

```
-o logfile
--output-file=logfile
    Log all messages to logfile.  The messages are normally reported to standard error.
```

I think the fix here is to just swap `wget` for `curl`.  Oh, Also, the URL we use is a redirect, so we'll have to pass in a `-L` flag so that curl follows the redirect too.

### PR DESCRIPTION
**Pull request by mrjones-plip**
Closes #2857

Changes proposed in this pull request:

- change link from `wget` to curl in `linux` update guide

How to test the feature manually:

1. run `wget -O freshrss.zip https://github.com/FreshRSS/FreshRSS/archive/1.15.3.zip` and make sure it works.
2. run `wget -o freshrss.zip https://github.com/FreshRSS/FreshRSS/archive/1.15.3.zip` and make sure it doesn't work ;)

Pull request checklist:

- [X] clear commit messages
- [X] code manually tested
- [] unit tests written (optional if too hard) 
- [X] documentation updated

[Additional information can be found in the documentation](https://github.com/FreshRSS/FreshRSS/tree/master/docs/en/developers/04_Pull_requests.md).

### CODE REVIEW

File: docs/en/admins/07_LinuxUpdate.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ cd /usr/share/FreshRSS/
 
 3. Download and unzip the update file
 ```
-wget -o freshrss.zip https://github.com/FreshRSS/FreshRSS/archive/1.15.3.zip
+curl -L -o freshrss.zip https://github.com/FreshRSS/FreshRSS/archive/1.15.3.zip
```
Comment (Frenzie) [inline docs/en/admins/07_LinuxUpdate.md]: As an aside, the typo was introduced by @marienfressinaud when changing from master.zip to x.y.z.zip. (I have no opinion for or against wget/curl.)

**Review by Frenzie — APPROVED** [review]

**Comment by Alkarex** [comment]
`wget` is more usual than `curl`.
For instance, `wget` is available by default in Ubuntu desktop, not `curl`
http://releases.ubuntu.com/eoan/ubuntu-19.10-desktop-amd64.manifest
So it is not a big deal, but I would favour `wget` a bit for such a documentation

File: docs/en/admins/07_LinuxUpdate.md (hunk 1 above)
Comment (Ths2-9Y-LqJt6) [reply docs/en/admins/07_LinuxUpdate.md]: > `wget` is more usual than `curl`.
> For instance, `wget` is available by default in Ubuntu desktop, not `curl`
> http://releases.ubuntu.com/eoan/ubuntu-19.10-desktop-amd64.manifest
> So it is not a bit deal, but I would favour `wget` a bit for such a documentation

@Alkarex  - All good! I just looked it up and it's `-O` for wget, so it'll make it a *2 char* PR ;)  Just a sec here...

**Comment by Frenzie** [comment]
Huh, I don't remember installing curl. But then wget it is. ;-)

**Comment by mrjones-plip** [comment]
> `wget` is more usual than `curl`.
> For instance, `wget` is available by default in Ubuntu desktop, not `curl`
> http://releases.ubuntu.com/eoan/ubuntu-19.10-desktop-amd64.manifest
> So it is not a bit deal, but I would favour `wget` a bit for such a documentation

@Alkarex  - All good! I just looked it up and it's `-O` for wget, so it'll make it a *2 char* PR ;)  Just a sec here...

**Comment by Alkarex** [comment]
@Frenzie  `curl` is available by default only in the server version of Ubuntu
http://releases.ubuntu.com/eoan/ubuntu-19.10-live-server-amd64.manifest

**Review by Alkarex — APPROVED** [review]

**Pull request merged**